src/client/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "types": ["vite/client", "node", "openfin", "jest"],
     "allowJs": false,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
yeah, I had to do this testing on command line .. guess we are at the mercy of old OpenFin libs, unless we can specifically exclude that case ?

Reply:
I think we can resolve this by upgrading our OF code as a separate tech debt remediation effort